### PR TITLE
Feature/form group component

### PR DESCRIPTION
**Pull request by kumardennis**

### CODE REVIEW

File: src/Components/FormGroup/component.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <Form.Group controlId={this.state.controlId}>
+                <Form.Label>{this.state.label}</Form.Label>
+                <Form.Control
+                    type={this.state.type}
+                    placeholder={this.state.placeholder}
+                />
+
+                <Form.Text className="text-muted">
+                    {this.state.muted !== undefined ? this.state.muted : ''}
+                    {/* why cant we use null? */}
```
Comment (edanchenkov) [inline src/Components/FormGroup/component.js]: As I wrote somewhere else, I prefer to use only `undefined` unless 3rd party lib or API returns "null" value. Imo working with `undefined` is easier than with `null`. Do you have any reasons to use `null`?

**Review by edanchenkov — COMMENTED** [review]
@kumardennis Could you add a comprehensive example to storybooks? E.g. how this component can look like in different context with different props/states. You can check here https://github.com/PlatformOfTrust/react-elements-of-trust/pull/37/files

File: src/Components/FormGroup/component.js (hunk 1 above)
Comment (kumardennis) [reply src/Components/FormGroup/component.js]: i have usually worked with null but its okay if its a preference issue :D i got no reasons for using it or not using it.

Comment (edanchenkov) [reply src/Components/FormGroup/component.js]: Yeah, I believe it comes from the fact that `typeof foo !== 'undefined'` was quite faster than any other kind of `foo !== null` or similar checks in the past. Right now maybe it is not as big issue as it was. But I have a habit now. 

You could find plenty of material on undefined vs null. I believe `undefined` is still more accepted than using `null`, but probably only on preference base nowadays.

Comment (kumardennis) [reply src/Components/FormGroup/component.js]: okay thanks

File: stories/index.js [hunk 2]
```diff
@@ -26,3 +27,13 @@ storiesOf('Button', module)
...
     .add('default', () => <FormGroup controlId={'formid'} muted={'this should be muted'} label={'Ask question'} type={'email'} placeholder={'placeholder text...'} text="Hello World!" />);
+
+storiesOf('Image', module).addParameters({
+    backgrounds: [
+        { name: 'light', value: '#fffef0' },
+        { name: 'dark', value: '#1e3b47' },
+        { name: 'purple', value: '#7955ff', default: true }
+    ],
+}).add('default', () => (
+    <Image src="https://i.pinimg.com/originals/aa/49/54/aa4954451f16847cb657e68e5f46538a.jpg"/>
```
Comment (edanchenkov) [inline stories/index.js]: Maybe add local image instead?

Comment (kumardennis) [reply stories/index.js]: Image component is already merged into master, its not made by me. Do you still want me to change the url of Image compoennt in all of my branches? im sure there can be an easier solution

Comment (edanchenkov) [reply stories/index.js]: You should submit only one pr using local image for image component. And fix all your branches where you use images to use local image.

**Review by victorv-sbp — APPROVED** [review]

**Pull request merged**